packages/plugins/i18n/server/src/bootstrap.ts

```diff
@@ -1,4 +1,5 @@
 import type { Schema } from '@strapi/types';
+import { isEqual } from 'lodash';
```

Comment:
Could you use the export from `lodash/fp` I think we try to favor the functional programming lodash. Heads up the function signature could be different.